person.rb

```diff
@@ -1,8 +1,11 @@
+require_relative 'nameable'
+
 class Person
```

Comment:
- [ ] Please try to make the person class inherit from nameable and create the `correct_name` method in the person class.